.github/workflows/build-deploy.yaml

```diff
@@ -35,15 +35,9 @@ jobs:
       shell: bash
       run: |
         set -x
-        if [[ "${{ github.ref }}" =~ ^refs/tags.* ]]
+        if [[ "${{ github.ref }}" =~ ^refs/heads/develop ]]
         then
-          echo "::set-output name=environment::production"
-        elif [[ "${{ github.ref }}" =~ ^refs/heads/staging ]]
-        then
-          echo "::set-output name=environment::staging"
-        elif [[ "${{ github.ref }}" =~ ^refs/heads/develop ]]
-        then
-          echo "::set-output name=environment::develop"
+          echo "::set-output name=environment::test"
```

Comment:
just curious if i'm getting it correct - deploying to testnet from develop?

Reply:
Yes, you are right